webpack.config.js

```diff
@@ -90,6 +90,11 @@ module.exports = (env, argv) => {
       ]
     },
     resolve: {
+      alias: {
+        // prevent duplicate react versions when npm linking lara-interactive-api
+        // cf. https://github.com/facebook/react/issues/13991#issuecomment-435587809
+        react: path.resolve(__dirname, './node_modules/react'),
```

Comment:
Nice you found a fix, that's what lara-interactive-api 0.4.0 release is now at pre.10. :wink: